packages/itinerary-body/src/ItineraryBody/place-row.tsx

```diff
@@ -54,7 +54,13 @@ export default function PlaceRow({
   // check the real-time arrival information for the next leg of their journey.
   const interline = !!(!isDestination && leg.interlineWithPreviousLeg);
   const hideBorder = interline || !legIndex;
-  const place = isDestination ? leg.to : leg.from;
+  const place = isDestination ? { ...leg.to } : { ...leg.from };
```

Comment:
Why was this change made?

Reply:
We need a copy of the place so that when we edit the vertex type on the next line it doesn't cause any side effects.